woocommerce/woocommerce-ios
# Display error feedback when submitting Shipping Label Address

Summary
==========
Fix issue #4909 by triggering a `Notice` whenever there's an issue with the Address submission, instead of just letting the `done` and `Use Address as Entered` button silent. The Android side currently can evaluate in details what's the issue to show inside the `Notice` view, but since the iOS form do warn what's wrong inside the View, I decided to keep the failure handling message simple and close to the text used on the Android side.

Screenshots
==========
| Android error handling | iOS error handling |
| ------------- | ------------- |
| ![Screenshot_20220427_202553](https://user-images.githubusercontent.com/5920403/165649339-25f25ea5-2809-4fc1-9af6-f28b9445bd38.png) | ![IMG_8883](https://user-images.githubusercontent.com/5920403/165649547-2f2fb48a-afe0-419a-b776-4b430df092a8.PNG) |

How to Test
==========
1. Make sure your Woo site supports the Shipping Label plugin and it's correctly configured to work with it
2. Select an Order with the status as `Processing` and select `Create Shipping Label`
3. At the `Create Shipping Label` view, select the `Continue` button under the `Ship From` Address info
4. Try to reproduce an Address issue, like an empty Phone number, and hit `done` or `Use Address as Entered`
5. Verify that an error Notice is now displayed instead of silently handling the address validation.

Update release notes:
